stf/src/saros/stf/shared/Constants.java

```diff
@@ -10,19 +10,9 @@
     CONCURRENTLY
```

Comment:
If the _enum_ is not used on the server part it should not be placed here.

Reply:
But these constants aren't STF client/server-specific, are they? The enum is basically only used in combination with `TypeOfCreateProject`, so it only seemed natural to define it in the same class. Where else should I put it?

Reply:
Guess it should be moved to the client. But this also includes many other stuff and so this is not the best idea to do this in this patch.